src/sqlancer/mysql/MySQLSchema.java

```diff
@@ -139,12 +139,25 @@ private static MySQLDataType getColumnType(String typeString) {
         case "mediumint":
         case "int":
         case "bigint":
+        case "integer":
             return MySQLDataType.INT;
         case "varchar":
         case "tinytext":
         case "mediumtext":
         case "text":
         case "longtext":
+        case "char":
```

Comment:
It seems none of these new data types is actually exercised, so this seems to be dead code.

Reply:
I think adding support for a currently unsupported data type would be generally valuable.